sdk/python/kfp/compiler/_data_passing_rewriter.py

```diff
@@ -157,20 +157,6 @@ def fix_big_data_passing(workflow: dict) -> dict:
 
     # Searching for parameter input consumers in container and resource templates
     for template in container_templates + resource_templates:
-        # Hack for volumes: Prior to Argo 2.3.0, it was only possible to add volumes globally on the workflow level instead of at template level.
```

Comment:
Since `volume_map` was only used in this "hack", it is no longer used. Maybe remove this line as well:
https://github.com/kubeflow/pipelines/blob/6587d36e910fa4b70a5eb80b11487cb522209c6a/sdk/python/kfp/compiler/_data_passing_rewriter.py#L31

Reply:
Done. Thanks.